topiary/commandline_args.py

```diff
@@ -28,6 +28,7 @@
 from .parsing_helpers import parse_int_list
 from .rna import (
     load_cufflinks_fpkm_dict,
```

Comment:
"gtf" and "tracking" are both file formats (sort of), but the latter is only associated with Cufflinks, so just saying "tracking" may be unclear. I guess I can always write "cufflinks-tracking-file". What do you think of that?